.github/workflows/ci.yaml

```
@@ -19,8 +19,14 @@ jobs:
           java-version: '21'
           distribution: 'temurin'
 
-      - name: Build
-        run: gradle classes
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
 
-      - name: Test
-        run: gradle test
\ No newline at end of file
+      - name: Compile
+        run: ./gradlew build
```

```suggestion
        run: ./gradlew classes
```
Build runs tests as well. We want here only compile sources.